src/components/buoy-telemetry/BuoyMap.vue

```diff
@@ -45,11 +45,11 @@ const props = defineProps<{
 }>();
 
 const annotatedSamples = computed(() => {
-  const domain = [-5, 27];
-  const sqrtScale = scaleSqrt().domain(domain).range([0.2, 0.7]);
+  const domain = [0, 45];
```

Comment:
Out of curiosity how are you determining these domain and range values? By inspection? Also maybe want to make the variable name more specific? `chlorophyll_domain`? 

And maybe also have
`const chlorophyll_ranage` and `const chlorophyll_colormap` variables abstracted? 

I think soon, you'll be having similar definitions for multiple measurement, and keeping them labeled and abstracted will make it simpler to follow

Reply:
The domain is based on the min/max of the chlorophyll values (0-60), with a clamp so anything over 45 just shows as the darkest color. The range I just did manual inspection to see what it looks like. The range determines the size of the circle marker in this case. 

If I abstract the variables, then would I remove chlorophyll from the name so that the selected variable could be swapped in?